frontend/express/public/javascripts/countly/vue/components/sidebar.js

```diff
@@ -0,0 +1,425 @@
+/* global app, jQuery, CV, Vue, countlyGlobal, _, Backbone*/
+
+(function(countlyVue, $) {
+
+    $(document).ready(function() {
+
+        var UsersMenu = countlyVue.views.create({
+            template: CV.T('/javascripts/countly/vue/templates/sidebar/users-menu.html'),
+            props: {
+                item: {
+                    type: Object
+                }
+            },
+            data: function() {
+                return {
+                    helpCenterLink: {
+                        isString: typeof countlyGlobal.usermenu.helpCenterLink === "string" ? countlyGlobal.usermenu.helpCenterLink : false,
+                        isBoolean: typeof countlyGlobal.usermenu.helpCenterLink === "boolean" && countlyGlobal.usermenu.helpCenterLink
+                    },
+                    documentationLink: {
+                        isString: typeof countlyGlobal.usermenu.documentationLink === "string" ? countlyGlobal.usermenu.documentationLink : false,
+                        isBoolean: typeof countlyGlobal.usermenu.documentationLink === "boolean" && countlyGlobal.usermenu.documentationLink
+                    },
+                    feedbackLink: {
+                        isString: typeof countlyGlobal.usermenu.feedbackLink === "string" ? countlyGlobal.usermenu.feedbackLink : false
+                    },
+                    featureRequestLink: {
+                        isString: typeof countlyGlobal.usermenu.featureRequestLink === "string" ? countlyGlobal.usermenu.featureRequestLink : false,
+                        isBoolean: typeof countlyGlobal.usermenu.featureRequestLink === "boolean" && countlyGlobal.usermenu.featureRequestLink
+                    }
+                };
+            }
+        });
+
+        var AnalyticsMenu = countlyVue.views.create({
+            template: CV.T('/javascripts/countly/vue/templates/sidebar/analytics-menu.html'),
+            mixins: [
+                countlyVue.container.dataMixin({
+                    "categories": "/sidebar/analytics/menuCategory",
+                    "menus": "/sidebar/analytics/menu",
+                    "submenus": "/sidebar/analytics/submenu"
+                })
+            ],
+            data: function() {
+                var apps = _.sortBy(countlyGlobal.apps, function(app) {
+                    return (app.name + "").toLowerCase();
+                });
+
+                apps = apps.map(function(a) {
```

Comment:
forEach would do the same thing as you just mutate the existing items in the array, rather than creating new ones.